Assets/Scripts/Spawning/ResourceSpawning/Structs/ResourceSpawnInfo.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using UnityEngine;
+
+namespace Examen.Spawning.ResourceSpawning.Structs
+{
+    [Serializable]
+    public struct ResourceSpawnInfo
+    {
+        [SerializeField] private GameObject _spawnResource;
+        [Range(0, 100)]
+        [SerializeField] private float _spawnChance;
+
+        public readonly GameObject SpawnResource => _spawnResource;
+        public readonly float SpawnChance => _spawnChance;
```

Comment:
Maybe simplify the names because now you need to call `ResourceSpawnInfo.SpawnResource` and `ResourceSpawnInfo.SpawnChange` but because the struct is already called `ResourceSpawnInfo` I would leave out the word "Spawn" from the variable names.

```suggestion
        public readonly GameObject Resource => _spawnResource;
        public readonly float Chance => _spawnChance;
```